### PR TITLE
Feature - Remove declarations if ASRU changes SOP

### PR DESCRIPTION
**Pull request by joefitter**
* Removed declaration if ASRU for create update and delete
* Fixed copy on delete role confirm page - submit to ASRU -> submit

### CODE REVIEW

**Comment by joefitter** [comment]
needs https://github.com/UKHomeOffice/asl-workflow/pull/190 to be deployed at the same time

File: pages/place/routers/confirm.js [hunk 1]
```diff
@@ -27,46 +27,49 @@ module.exports = settings => {
...
-    },
-    editAnswers: (req, res, next) => {
-      delete req.session.form[req.model.id].validationErrors;
-      return res.redirect(req.buildRoute(settings.page));
-    },
-    cancelEdit: (req, res, next) => {
-      return res.redirect(req.buildRoute('place.list'));
-    }
-  }, settings)));
+    }, settings))(req, res, next);
```
Comment (lennym) [inline pages/place/routers/confirm.js]: I really hate the pattern of middlewares that call other middlewares.

Could we set `req.requiresDeclaration` in a previous step, and get rid of that?

Comment (lennym) [reply pages/place/routers/confirm.js]: Alternatively make `requiresDeclaration` a function that takes the request as an argument and returns a `Boolean`

Comment (joefitter) [reply pages/place/routers/confirm.js]: don't be a hater

Comment (joefitter) [reply pages/place/routers/confirm.js]: also yes, although the other 2 corresponding PRs do it this way, so I guess you want them updating too?

File: pages/role/remove/content/confirm.js [hunk 2]
```diff
@@ -12,7 +12,7 @@ module.exports = merge({}, baseContent, {
     declaration1: `Applicant authority, conflict of interests and PELh authority declaration text to go here`
   },
   buttons: {
-    submit: 'Submit to ASRU'
+    submit: 'Submit'
```
Comment (lennym) [inline pages/role/remove/content/confirm.js]: 🎉

File: pages/place/index.js [hunk 3]
```diff
@@ -6,7 +6,10 @@ module.exports = settings => {
   const app = Router({ mergeParams: true });
 
   app.param('placeId', (req, res, next, placeId) => {
-    return req.api(`/establishment/${req.establishmentId}/place/${placeId}`)
+    const params = req.path.match(/delete\/success$/)
+      ? { query: { withDeleted: true } }
+      : {};
```
Comment (lennym) [inline pages/place/index.js]: I'd be tempted to just hard-code this.

**Pull request merged**